server/getDirections.js

```diff
@@ -0,0 +1,62 @@
+var GoogleMapsLoader = require('google-maps'); 
+
+// this fn will take two points and spit out google directions
+ 
+// To use directions in the Google Maps JavaScript API, create an object of
+// type DirectionsService and call DirectionsService.route() to initiate a request
+// to the Directions service, passing it a DirectionsRequest object literal 
+// containing the input terms and a callback method to execute upon receipt of the response.
+
+// DirectionsRequest example:
+// {
+//   origin: LatLng | String | google.maps.Place,
+//   destination: LatLng | String | google.maps.Place,
+//   travelMode: TravelMode,
+//   transitOptions: TransitOptions,
+//   drivingOptions: DrivingOptions,
+//   unitSystem: UnitSystem,
+//   waypoints[]: DirectionsWaypoint,
+//   optimizeWaypoints: Boolean,
+//   provideRouteAlternatives: Boolean,
+//   avoidHighways: Boolean,
+//   avoidTolls: Boolean,
+//   region: String
+// }
+
+var directionsService = new google.maps.DirectionsService();
+
+// userlocation will be an object with a latitude and longitude property (can change later)
+// destination will be an object with a latitude and longitude property (can change later)
+// closestStation is an object from our db containing the bike station that is cloest to the user
+  // for MVP, we are taking only the CLOSEST station to the user, and the CLOSEST
+  // station to the destination and assuming those as our bike stations
+// destClosestStation is an object from our db that represents the closest station 
+// to the user's destination
+var getDirections = function (userLocation, destination, userClosestStation, destClosestStation) {
+  var userLocation = new google.maps.LatLng(userLocation.latitude, userLocation.longitude);
```

Comment:
remove vars from userLocation and destination. They're already initialized as variables when passed in as params.